templates/security/profile.html.twig

```diff
@@ -10,7 +10,10 @@
             <span class="badge badge-secondary">{{ role|lower }}</span>
         {% endfor %}
         <div class="mb-2">
-            <span class="badge badge-primary">Website</span> <a href="{{ user.homepage }}"></a>{{ user.homepage }}
+            <span class="badge badge-primary">Website</span> <a href="{{ user.homepage|escape }}"></a>{{ user.homepage }}
```

Comment:
```suggestion
            <span class="badge badge-primary">Website</span> <a href="{{ user.homepage }}"></a>{{ user.homepage }}
```
Symfony standartiškai uždeda `htmlspecialchars`, todėl nebūtina papildomai raštyti `|escape` filtro.

Plačiau:
 * https://symfony.com/doc/current/templating/escaping.html
 >  If you're using Twig, output escaping is on by default and you're protected.